lib/compress/zstd_compress.c

```diff
@@ -202,29 +197,109 @@ size_t ZSTD_sizeof_CStream(const ZSTD_CStream* zcs)
 /* private API call, for dictBuilder only */
 const seqStore_t* ZSTD_getSeqStore(const ZSTD_CCtx* ctx) { return &(ctx->seqStore); }
 
-static ZSTD_parameters ZSTD_getParamsFromCCtx(const ZSTD_CCtx* cctx) { return cctx->appliedParams; }
-
 /* older variant; will be deprecated */
+/* Both requested and applied params need to be set as this function can be
```

Comment:
Question : is it time to consider retiring it ?

Reply:
Did it only still exist to support zstdmt? It's the only way to set forceWindow/forceRawDict with the old api (I think)

Reply:
Yes, it was created for `zstdmt`.
You may now be able to transmit these parameters with your new `ZSTD_CCtx_params` structure.